docs/BOOTING.md

```diff
@@ -147,3 +147,13 @@ decide to implement IMGA/IMGB partition selection directly in u-boot to support
 up until this point, it seems that all the boards that are modern enough to support EVE's virtualization requirements are also modern
 enough to support UEFI environment directly (even for HiKey where we're currently using u-boot as a stop gap measure the proper
 [UEFI implementation](https://github.com/pftf/RPi4) is very much in the works).
+
+## Booting Raspberry Pi with netboot
+
+In order to boot RPi 4 from network you should modify bootloader configuration as described [here](https://www.raspberrypi.org/documentation/hardware/raspberrypi/bcm2711_bootloader_config.md).
+You should modify BOOT_ORDER to one that uses NETWORK mode (for example `0xf12`).
```

Comment:
Actually, I don't include anything related to eeprom, because it can break RPi accidentally. Also, tools for modification of eeprom recommended to use from up-to-date Raspbian.

Reply:
Fair enough, but them please update documentation with example set of steps that you yourself went through (and what platform you ran them on). Like:

```
apt install rpi-eeprom -y
...
rpi-eeprom-update -d -f ./pieeprom-new.bin
```

Or whatever was the process

Reply:
Done. I put two variants: netboot and boot from usb.